inc/class-core-sitemaps-provider.php

```diff
@@ -136,6 +136,31 @@ public function get_url_list( $page_num, $type = '' ) {
 
 		$url_list = array();
 
+		/*
+		 * Add a URL for the homepage in the pages sitemap.
+		 * Shows only on the first page if the reading settings are set to display latest posts.
+		 */
+		if ( 'page' === $type && 1 === $page_num && 'posts' === get_option( 'show_on_front' ) ) {
+			// Assumes the homepage last modified date is the same as the most recent post.
+			$last_modified = get_posts(
+				array(
+					'post_type'              => 'post',
+					'posts_per_page'         => '1',
```

Comment:
Is there a reason this value is being passed as a string? The WP_Query docs expect an integer here.

Reply:
Good catch.

Reply:
Simplified the args here in `8f596b2`.